main.go

```diff
@@ -31,19 +31,26 @@ func init() {
 	}
 }
 
-func main() {
-	err := app.Flag.Parse(os.Args[1:])
+func runArgs(argsStr []string) error {
```

Comment:
I would call this function just `run` and its argument `args`.

Reply:
Done. I had to rename the args variable in the method to appArgs to avoid conflict in naming.